webapp/commonComponents/tooltip.scss

```diff
@@ -42,6 +42,10 @@
     border: 2px solid rgba($red, 0.6);
   }
 
+  .cycle {
+    /* Without !important, other border stuff is more specific! */
+    border: 2px solid rgba($red, 0.6) !important;
+  }
```

Comment:
Yeah I'll move it to SurveyInfoView

In this case !important is 'necessary' because the normal border CSS selector is more specific (3 or 4 CSS classes specified).
Well, I could figure out some alternative but then again, I'm not sure if it's worth the time to "fix" this very marginal exception (errors are "important", after all, so I believe it should not manifest as a problem, probably ever)

Reply:
Turns out this was resolvable with minimum fuss now by moving it immediately below the default 
 border style declaration:)

Reply:
@minotogna you found a nice solution, thanks! I didn't see it at the time.